views/salesEmployee/create.njk

```diff
@@ -0,0 +1,104 @@
+{% extends "templates/layout.njk" %}
```

Comment:
the templace is create.njk, create what?

Reply:
salesEmployee/create.njk